hknweb/candidate/admin.py

```diff
@@ -93,9 +93,15 @@ def set_visible(self, request, queryset):
 
     def set_invisible(self, request, queryset):
         queryset.update(visible=False)
-        
+
     set_invisible.short_description = "Set selected as invisible"
 
+class CandidateFormAdmin(admin.ModelAdmin):
+    fields = ['name', 'link', 'duedate']
+    list_display = ['name', 'link', 'duedate']
+    list_filter = ['name', 'link', 'duedate']
```

Comment:
Don't think there's a need to filter by link. Probably don't need to filter by name either since they should be unique most of the time.